eegnb/experiments/Experiment.py

```diff
@@ -0,0 +1,168 @@
+""" 
+Initial run of the Experiment Class Refactor base class
+
+Specific experiments are implemented as sub classes that overload a load_stimulus and present_stimulus method
+
+Running each experiment:
+obj = VisualP300({parameters})
+obj.run()
+"""
+
+from abc import ABC, abstractmethod
+from psychopy import prefs
+#change the pref libraty to PTB and set the latency mode to high precision
+prefs.hardware['audioLib'] = 'PTB'
+prefs.hardware['audioLatencyMode'] = 3
```

Comment:
Are these safe to set globally? @JohnGriffiths 

Feels like we set them like this in more places, in which case they would conflict.

Edit: Indeed, but looks like they all set the same values? Should prob be consolidated. All hits: https://github.com/NeuroTechX/eeg-notebooks/search?q=prefs.hardware